browse/controllers/tb_page/__init__.py

```diff
@@ -14,10 +15,10 @@
                                      get_trackback_ping
 from browse.controllers import check_supplied_identifier
 from browse.domain.identifier import Identifier, IdentifierException
-from browse.services.document import metadata
-from browse.services.document.metadata import AbsException, \
+from browse.services.documents import get_doc_service
+from browse.services.documents.base_documents import AbsException, \
     AbsNotFoundException
-from browse.services.search.search_authors import queries_for_authors, \
+from browse.formating.search_authors import queries_for_authors, \
```

Comment:
```suggestion
from browse.formatting.search_authors import queries_for_authors, \
```
formating -> formatting (multiple places)